addon-test-support/qunit-configuration.js

```diff
@@ -0,0 +1,9 @@
+import QUnit from 'qunit';
```

Comment:
When the app is using ember-auto-import, this line probably only works by accident.

That is, ember-auto-import won't parse this file to see that qunit is needed, because ember-auto-import is not a dependency of ember-qunit. So this import will fail at runtime, unless some other package has already caused qunit to be auto-imported.

In the common case, the app's test suite will have at least one place that imports from qunit, making this work. But it will fail very confusingly if they, say, have no tests and therefore have no place that imports qunit.

I think it would be simpler to make ember-qunit depend on ember-auto-import and always auto-import qunit. That would also allow you to delete all the code that decides whether to conditionally run `app.import`. For the compatibility case of an app that doesn't have auto-import, it would still work because the app could rely on the side effect of ember-qunit importing qunit causing it to be defined (this is the reverse of the bug I'm pointing out here -- I think it's fine if the addon always does an import and the app relies on that by side effect, whereas I don't think the addon should rely on a side effect of the app doing an import).

Reply:
> I think it would be simpler to make ember-qunit depend on ember-auto-import and always auto-import qunit. That would also allow you to delete all the code that decides whether to conditionally run app.import.

Thank you for reviewing! I'll test that now...

Reply:
Updated in 257556d